src/components/FeatureViews/AdminPortal/UnpaidSessionCard.js

```diff
@@ -27,46 +29,54 @@ const UnpaidSessionCard = ({unpaidStudent}) => {
-                        }}>
-                        {initials(
-                            student.first_name, student.last_name
-                        )}
-                    </Avatar>
-                </Grid>
-            </CardMedia>
-            <CardContent className="unpaid-details">
-                <Typography className="unpaid-student-name">
-                    {student.name}
-                </Typography>
-                <Typography className="unpaid-role-label">
-                    Student
-                </Typography>
-                <Typography>
-                    Payment Status: <span
-                        className="unpaid-status"
-                        style={{
-                            "backgroundColor": adminUtils.statusColor[
-                                unpaidStudent.sessions_left
-                            ],
-                        }}>
-                        {unpaidStudent.sessions_left}
-                    </span>
-                    <br />
-                    ${amtDue}
-                    <br />
-                    {course.title}
-                </Typography>
-            </CardContent>
+            <CardActionArea
+            component = {NavLink}
+            to={`/accounts/students/${studentID}/${courseID}`}
+            >
+                <CardMedia>
+                    <Grid
+                        className="unpaid-avatar-container"
+                        container>
+                        <Avatar
+                            className="unpaid-avatar"
+                            style={{
+                                "backgroundColor": stringToColor(student.name),
+                            }}>
+                            {initials(
+                                student.first_name, student.last_name
+                            )}
+                        </Avatar>
+                    </Grid>
+                </CardMedia>
+                <CardContent className="unpaid-details">
+                    <Typography className="unpaid-student-name">
+                        {student.name}
+                    </Typography>
+                    <Typography className="unpaid-role-label">
+                        Student
+                    </Typography>
+                    <Typography className="unpaid-status-info">
+                        Payment Status: <span
+                            className="unpaid-status"
+                            style={{
+                                "backgroundColor": adminUtils.statusColor[
+                                    unpaidStudent.sessions_left
+                                ],
+                            }}>
+                            {unpaidStudent.sessions_left}
+                        </span>
+                        <br />
+                        Amount Due: ${amtDue}
+                        <br />
+                        {course.title}
```

Comment:
Wrap `course.title` in its own `<Typography>` tag. Give the tag a "nowrap" attribute and set it to true so that this will be an ellipses instead of wrapping on to the next line/overflowing [Typography documentation](https://material-ui.com/api/typography/). 

Then around this new `<Typography>` tag, wrap a `<Tooltip/>` that displays the entire name of the course on hover [Tooltip documentation](https://material-ui.com/api/tooltip/)

Currently still looks like this for long course names: 
![Screen Shot 2020-04-04 at 2 39 01 PM](https://user-images.githubusercontent.com/12959959/78461938-1cd88780-7682-11ea-8c82-5086c109e4e5.png)